drghs-worker/maintnerd/api/v1beta1/issue_service.go

```diff
@@ -236,16 +240,17 @@ func (s *IssueServiceV1) GetIssue(ctx context.Context, r *drghs_v1.GetIssueReque
 			return nil
 		}
 
-		return repo.ForeachIssue(func(issue *maintner.GitHubIssue) error {
-			if r.Name == getIssueName(repo, issue) && !issue.NotExist {
-				re, err := makeIssuePB(issue, repo.ID(), r.Comments, r.Reviews, r.FieldMask)
-				if err != nil {
-					return err
-				}
-				issueResp = re
-			}
+		issue := repo.GetIssue(issueID)
+		if issue == nil {
```

Comment:
don't also need to check if the issue is NotExist?

Reply:
Yeah.  @BrennaEpp and I caught that during a review earlier today. Will fix.